design-reference/docs/azure-simple.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Azure Simple
+draft: true
```

Comment:
Can you elaborate on what this field controls?

Reply:
`draft` will keep the page from being rendered to the live site. It will default to `true` and should be changed to `false` by the PR approver.

Reply:
Where in the PR cycle does this happen? Does the reviewer need to commit a change?

Reply:
Sorry, not changed by the PR approver--changed automatically in the CD pipeline after PR approval.